### PR TITLE
Fix rubygems/bundler labelling

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
Current labelling has false negatives, for example, if fails to label with "Rubygems" PRs changing `lib/rubygems.rb`.

Getting the workflow changes right is tricky. We should probably rename workflows in `rubygems/rubygems` to follow some convention, for example, name them `rubygems-<whatever>.yml` if there are specific to `rubygems`, `bundler-<whatever>.yml` if they are specific to `bundler`, or differently if they are common. I believe we plan to do this rename anyways.

@bronzdoc Thoughts?

### CODE REVIEW

**Comment by deivid-rodriguez** [comment]
I standarized workflow file naming in `rubygems/rubygems`, I believe this should now be correct?

File: webhook.rb [hunk 1]
```diff
@@ -33,6 +39,18 @@ class App < Sinatra::Base
     end
   end
 
+  def self.common_file?(file)
+    file =~ /\.github\/workflows\// && file !~ /\.github\/workflows\/.*-(rubygems|bundler)\.yml/
+  end
+
+  def self.bundler_file?(file)
+    file =~ /bundler\// || file == ".rubocop_bundler.yml" || file = /\.github\/workflows\/.*-bundler\.yml/
```
Comment (bronzdoc) [inline webhook.rb:47]: We are safe here unless we plan to add `lib/bundler.rb` 😆

**Review by bronzdoc — APPROVED** [review]

**Comment by bronzdoc** [comment]
Thanks @deivid-rodriguez! looks good to me!

File: webhook.rb (hunk 1 above)
Comment (deivid-rodriguez) [reply webhook.rb:47]: To rubygems? I don't think that will happen any time soon, altough it's an idea for a clean path for merging the libraries :smile:.

**Comment by deivid-rodriguez** [comment]
Thanks @bronzdoc!!

**Pull request merged**